netlify.toml

```diff
@@ -0,0 +1,19 @@
+# “command” is your build command.
+# “publish” is the directory to publish (relative to the root of your repo).
+[build]
+  command = "CI=true npm run test && npm run build"
+  publish = "build"
+
+# Deploy Preview context: All deploys generated from a pull/merge request
+# will inherit these settings.
+[context.deploy-preview.environment]
+  REACT_APP_CLIENT_ID = "711954789234.738124914208"
+# Not so secret after all 
+  REACT_APP_CLIENT_SECRET = "a6b2dcbfeff77561afde34eea60908cd"
```

Comment:
Shouldn't we use an env variable here instead of our API secret?

Reply:
We tried saving it in env variables but netlify.toml file can't read from netlify's environment variable. 

Any how, these are just for the staging environment. 
We are storing production variable in env.